frontend/packages/citizen-frontend/src/localization/sv.tsx

```diff
@@ -897,13 +875,13 @@ const sv: Translations = {
         secondGuardianInfo:
           'Den andra vårdnadshavarens information hämtas automatiskt från befolkningsinformationssystemet.',
         secondGuardianNotFound:
-          'Baserat på information från VTJ har barnet ingen andra vårdnadshavare',
+          'Baserat på information från befolknngsdata har barnet ingen andra vårdnadshavare',
```

Comment:
I guess exactly correct translation for väestötietojärjestelmä would be Befolkningsdatasystemet. But I guess it doesn't matter here.